src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-field.html

```diff
@@ -8,47 +9,45 @@
     <input type="search" placeholder="Search" class="form-control" id="search-bar-input" autofocus />
 
     {% if theme_use_meilisearch.index_uids %}
-      <select id="indexUidSelector" class="index-select">
-      {% for index_uid, index_name in theme_use_meilisearch.index_uids.items() %}
-        <option value="{{ index_uid }}">{{ index_name }}</option>
-      {% endfor %}
-      </select>
-    {% endif %}
+        <select id="indexUidSelector" class="index-select">
+          {% for index_uid, index_name in theme_use_meilisearch.index_uids.items() %}
+            <option value="{{ index_uid }}">{{ index_name }}</option>
+          {% endfor %}
+        </select>
+      {% endif %}
     <i class="fa-solid fa-magnifying-glass"></i>
     <!-- Include the MeiliSearch JavaScript library for the search bar -->
     <script src="https://cdn.jsdelivr.net/npm/docs-searchbar.js@latest/dist/cdn/docs-searchbar.min.js"></script>
+
+    {% if not theme_use_meilisearch.host %}
+      {% set HOST_URL = "https://backend.search.pyansys.com/" %}
+    {% else %}
+      {% set HOST_URL = theme_use_meilisearch.host %}
+    {% endif %}
     
-      <!-- If no credentials provided, use the default ones  -->
-      {% if not theme_use_meilisearch.host and not theme_use_meilisearch.api_key %}
-        {% set HOST_URL = "https://search.pyansys.com" %}
-        {% set API_KEY = "A1Qxly2sls6g1rb1sIKEVEtE7R1UsHad0VgqnvKhD5nhBNf5de" %}
-      {% else %}
-        {% set HOST_URL = theme_use_meilisearch.host %}
-        {% set API_KEY = theme_use_meilisearch.api_key %}
-      {% endif %}
       <script>
       // Initialize the MeiliSearch bar with the given API key and host
       // inspect the first value of index UID as default
-      let theSearchBar = docsSearchBar({
-        hostUrl: "{{ HOST_URL }}",
-        apiKey: "{{ API_KEY }}",
-        indexUid: "{{ theme_use_meilisearch.index_uids.items()|first|first }}",
-        inputSelector: '#search-bar-input, #indexUidSelector',
-        enableDarkMode: 'auto',
-        debug: true // Set debug to true if you want to inspect the dropdown
+      var theSearchBar = docsSearchBar({
+          hostUrl: "{{ HOST_URL }}",
+          apiKey: "{{ API_KEY }}",
+          indexUid: "{{ theme_use_meilisearch.index_uids.items()|first|first }}",
+          inputSelector: '#search-bar-input',
+          enableDarkMode: 'auto',
+          debug: true // Set debug to true if you want to inspect the dropdown
```

Comment:
@Revathyvenugopal162 I am assuming we will set it to `false` later on.

Reply:
yes, may be after the inspection .